fxa-auth-db-server/test/backend/remote.js

```diff
@@ -718,6 +722,7 @@ module.exports = function(cfg, makeServer) {
             assert.equal(devices[0].callbackURL, '', 'callbackURL is correct')
             assert.equal(devices[0].callbackPublicKey, user.device.callbackPublicKey, 'callbackPublicKey is correct')
             assert.equal(devices[0].callbackAuthKey, user.device.callbackAuthKey, 'callbackAuthKey is correct')
+            assert.equal(devices[0].pushEndpointExpired, user.device.pushEndpointExpired, 'pushEndpointExpired is correct')
```

Comment:
Should this return `null` or be coerced to `false`?

Reply:
The db column definition says "not null" so it may be worth explicitly checking that it's `=== false` here rather than just a falsy null.